test/Microsoft.AspNet.Mvc.FunctionalTests/SerializableErrorTests.cs

```diff
@@ -64,5 +64,32 @@ public class SerializableErrorTests
             var responseData = await response.Content.ReadAsStringAsync();
             Assert.Equal(expectedXml, responseData);
         }
+
+        [Theory]
+        [InlineData("application/xml-xmlser")]
+        [InlineData("application/xml-dcs")]
+        public async Task IsReturnedInExpectedFormat(string acceptHeader)
+        {
+            // Arrange
+            var server = TestServer.Create(_services, _app);
+            var client = server.CreateClient();
+            var input = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
+                "<Employee xmlns=\"http://schemas.datacontract.org/2004/07/XmlFormattersWebSite.Models\">" +
+                "<Id>2</Id><Name>foo</Name></Employee>";
+            var expected = "<Error><employee.Id>The field Id must be between 10 and 100.</employee.Id>" +
+                            "<employee.Name>The field Name must be a string or array type with a minimum " + 
+                            "length of '15'.</employee.Name></Error>";
+            var request = new HttpRequestMessage(HttpMethod.Post, "http://localhost/SerializableError/CreateEmployee");
+            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse(acceptHeader));
+            request.Content = new StringContent(input, Encoding.UTF8, "application/xml-dcs");
```

Comment:
This isn't using the header. I think this test has some issues since xml-ser will generate different output.